### PR TITLE
Fixe weights is None for detr & Deformable

### PR DESCRIPTION
**Pull request by thibo73800**
In this pull request:
- Fixe weights is None for detr & Deformable

### CODE REVIEW

**Comment by Johansmm** [comment]
@thibo73800, Is it possible to remove the double condition "`if weights is not None"` for both cases? I mean, in the internal conditions, I think that is not necessary ask again `if weights is not None`

File: alonet/deformable_detr/deformable_detr.py [hunk 1]
```diff
@@ -167,14 +168,15 @@ def __init__(
...
-        if weights is not None and (
-            weights in ["deformable-detr-r50", "deformable-detr-r50-refinement"]
-            or ".pth" in weights
-            or ".ckpt" in weights
-        ):
-            alonet.common.load_weights(self, weights, device, strict_load_weights=strict_load_weights)
-        else:
-            raise ValueError(f"Unknown weights: '{weights}'")
+        if weights is not None:
+            if weights is not None and (
```
Comment (Johansmm) [inline alonet/deformable_detr/deformable_detr.py:172]: Maybe is not necessary make the question "weights is not None" again

File: alonet/detr/detr.py [hunk 2]
```diff
@@ -83,10 +83,11 @@ def __init__(
         if device is not None:
             self.to(device)
 
-        if weights is not None and (weights == "detr-r50" or ".pth" in weights or ".ckpt" in weights):
-            alonet.common.load_weights(self, "detr-r50", device, strict_load_weights=strict_load_weights)
-        else:
-            raise ValueError(f"Unknown weights: '{weights}'")
+        if weights is not None:
```
Comment (Johansmm) [inline alonet/detr/detr.py:86]: Same here

**Review by Johansmm — CHANGES_REQUESTED** [review]

File: alonet/deformable_detr/deformable_detr.py (hunk 1 above)
Comment (thibo73800) [reply alonet/deformable_detr/deformable_detr.py]: yes, done

File: alonet/detr/detr.py (hunk 2 above)
Comment (thibo73800) [reply alonet/detr/detr.py:86]: yes, done

**Pull request merged**